From 26c4f23fdf1e50dc1b46bcf1cf695aa5bbab2b14 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Tue, 30 May 2023 00:02:53 +0200
Subject: [PATCH 402/406] Revert "Bluetooth: hci_h5: Add support for binding
 RTL8723CS with device tree"

This reverts commit f88fedc0c70499734cf54521d9d06fa20540cb59.
---
 drivers/bluetooth/hci_h5.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/bluetooth/hci_h5.c b/drivers/bluetooth/hci_h5.c
index 78f35aec8584..6455bc4fb5bb 100644
--- a/drivers/bluetooth/hci_h5.c
+++ b/drivers/bluetooth/hci_h5.c
@@ -1102,8 +1102,6 @@ static const struct of_device_id rtl_bluetooth_of_match[] = {
 	  .data = (const void *)&h5_data_rtl8723bs },
 	{ .compatible = "realtek,rtl8723ds-bt",
 	  .data = (const void *)&h5_data_rtl8723bs },
-	{ .compatible = "realtek,rtl8723cs-bt",
-	  .data = (const void *)&h5_data_rtl8723bs },
 #endif
 	{ },
 };
-- 
2.34.1

